docs/content/installation/install_compatibility_requirements.md

```diff
@@ -17,15 +17,15 @@ The CSI driver requires the following ports to be opened on the worker nodes OS
 
         Port 7778
 
- -   **IBM Spectrum® Virtualize family includes IBM® SAN Volume Controller and IBM FlashSystem® family members that are built with IBM Spectrum® Virtualize (including FlashSystem 5xxx, 7200, 9100, 9200, 9200R)**
+ -   **IBM Spectrum® Virtualize family includes IBM® SAN Volume Controller and IBM FlashSystem® family members that are built with IBM Spectrum® Virtualize (including FlashSystem 5xxx, 7xxx, 9xxx)**
 
         Port 22
 
  -   **DS8000® family systems**
 
       Port 8452
 
-Complete these steps for each worker node in Kubernetes cluster to prepare your environment for installing the CSI (Container Storage Interface) driver.
+Complete these steps prepare your environment for installing the CSI (Container Storage Interface) driver.
```

Comment:
```suggestion
Complete these steps to prepare your environment for installing the CSI (Container Storage Interface) driver.
```

Reply:
this is being handled in PR https://github.com/IBM/ibm-block-csi-driver/pull/475